docs/usecases/datastorage_for_institutions.rst

```diff
@@ -128,25 +163,254 @@ where the data are located.
+   that GitHub/GitLab hosts are bare Git repositories, but that it appears as if
+   repositories on GitHub or GitLab -- unlike the bare repositories in a RIA
+   store that you have seen -- retain their directory hierarchy and their file
+   names, or that edits directly within GitHubs interface are possible, as if
+   those repositories would actually have a working tree:
+   GitHub's/GitLab's web-interface creates a *representation* of the repository content
+   and a *temporary* working tree to modify files directly from the web UI that
+   enables browsing and modifying file content. If you would push a bare
+   repository from a RIA store to GitHub/GitLab, it would also be represented with the
+   file and directory hierarchy of the original dataset.
+
+   Apart from enabling collaboration (a bare repository is required if you want to push to
+   it at all times), bare repositories have further advantages: Because they
+   do not contain a working tree or checkout, the repositories are leaner, and it
+   is easier for administrators to perform garbage collections and maintenance.
+   If you are interested in finding out more, you can read up on bare repositories
+   and how to use them
+   `here <https://git-scm.com/book/en/v2/Git-on-the-Server-Getting-Git-on-a
+   -Server>`_.
+
+
+Inside of the bare :term:`Git` repository, the ``annex`` directory -- just as in
+any standard dataset or repository -- contains the dataset's keystore (object
+tree) under ``annex/objects`` [#f2]_. In conjunction, keystore and bare Git
+repository are the original dataset -- just differently represented, with no
+*working tree*, i.e., directory hierarchy that exists in the original dataset,
+and without the name it was created under, but stored under its dataset ID.
+
+If necessary, the keystores can be (compressed) `7zipped <https://www.7-zip.org/>`_
+(``archives/``), either for compression gains, or for use on HPC-systems with
+`inode <https://en.wikipedia.org/wiki/Inode>`_ limitations.
+Despite being 7zipped, those archives could be indexed and support
+relatively fast random read access. Thus, the entire key store be put into an
+archive, re-using the exact same directory structure, and remains fully
+accessible while only using a handful of inodes, regardless of file number and size.
+
+On an infrastructural level, a RIA store is fully self-contained, and is a plain
+file system storage, not a database. It can be set up on any infrastructure that
+a dataset can be created on, with only few additional software requirements.
+
+.. findoutmore:: Software Requirements
+
+   .. todo::
+
+      @mih, @bpoldrack: please check!
+
+   - git-annex version 7.20 or newer
+   - DataLad version 0.12.3 (or later)
+   - Server side: 7z needs to be in the path.
+   - Only relevant for the application at hand: The ``cfg_inm7`` run procedure as provided with
+     ``pip install git+https://jugit.fz-juelich.de/inm7/infrastructure/inm7-datalad.git``
+
```

Comment:
@mih, @bpoldrack: what are current software requirements to create or use a RIA store?
Is the cfg_inm7 procedure still used?